src/ui/Card/Card.stories.jsx

```diff
@@ -28,179 +37,145 @@ export default {
-        ))}
-    </Card>
+      ) : (
+        ''
+      )}
+
+      {/* Decription */}
+      <Card.Description>{card.description}</Card.Description>
+    </Card.Content>
+
+    {/* Additional links */}
+    {card.links &&
+      card.links.map((item, index) => (
+        <Card.Content extra key={index}>
+          <a href={card.href}>{item.linkName}</a>
+        </Card.Content>
+      ))}
+  </Card>
+);
+
+const CardImage = (card) => (
+  <div className="image-with-title">
+    <Image
+      as="a"
+      href={card.href}
+      src={card.imgUrl}
+      wrapped
+      ui={false}
+      alt="card image"
+    />
+    <div className="gradient">
+      <Card.Header>
+        <a href={card.href}>{card.title}</a>
+      </Card.Header>
+    </div>
+  </div>
+);
+
+const MetaRight = (args) => (
+  <span className="bold text-right">{args.metaRight}</span>
+);
+
+const Template = (args) => (
+  <Container>
+    {args.cards.map((card, index) => (
+      <CardTemplate
+        variant={args.variant}
+        titleOnImage={args.titleOnImage}
+        inverted={args.inverted}
+        card={card}
+        key={index}
+      />
+    ))}
   </Container>
 );
 
 export const Default = Template.bind({});
 Default.args = {
+  titleOnImage: false,
   variant: 'default',
   inverted: false,
-  imgUrl:
-    'https://www.eea.europa.eu/media/pictures/european-environment-agency-building-with/image_large',
-
-  meta: 'Article',
-  metaRight: '30/07/2022',
-  title: 'Lorem Ipsum',
-  description:
-    'Leo fermentum sollicitudin suspendisse iaculis feugiat. Eget tellus blandit aenean mattis.',
-  hasImage: true,
-  href: '/#',
-  links: null,
-  fluid: false,
-};
-Default.argTypes = {
-  imgUrl:
-    'https://www.eea.europa.eu/media/pictures/european-environment-agency-building-with/image_large',
-  hasImage: {
-    description: 'true if card contains image',
-    table: {
-      type: {
-        summary: 'boolean',
-      },
-      defaultValue: { summary: true },
-    },
-  },
-  fluid: {
-    description: 'take up the width of its container',
-    table: {
-      type: {
-        summary: 'boolean',
-      },
-      defaultValue: { summary: false },
-    },
-  },
-  meta: {
-    description: 'Card metadata',
-    table: {
-      category: 'Card content',
-      type: { summary: 'string' },
-      defaultValue: { summary: ' "" ' },
-    },
-  },
-  metaRight: {
-    description: 'Card right metadata',
-    table: {
-      category: 'Card content',
-      type: { summary: 'string' },
-      defaultValue: { summary: ' "" ' },
-    },
-  },
-  title: {
-    description: 'card header',
-    table: {
-      category: 'Card content',
-      type: { summary: 'string' },
-      defaultValue: { summary: ' "" ' },
-    },
-  },
-  description: {
-    description: 'card main content',
-    table: {
-      category: 'Card content',
-      type: { summary: 'string' },
-      defaultValue: { summary: ' "" ' },
+  cards: [
+    {
+      title: 'Lorem Ipsum',
```

Comment:
@tarantilis  please use some real names for the cards such of the names found on the frontpage of the demo:
State of Europe's environment
Nature
Health
Climate
Economy and resources
Sustainability